benchmarking/benchmark.py

```diff
@@ -54,49 +54,101 @@ def get_times_skfp(X: np.ndarray, transformer_cls: type, **kwargs) -> np.ndarray
     return np.array(result).reshape((len(N_CORES), N_SPLITS))
 
 
-def save_results(
+LEGAL_PLOT_TYPES = ["time", "speedup", "fprints_per_second"]
+
+
+def save_plots(
     n_molecules: int,
     times: np.ndarray,
     title: str = "",
     save: bool = True,
```

Comment:
Why `save` parameter in `save_plots` function? Either it always saves, or change function name, e.g. `make_plot`

Reply:
Also it's always a singular plot, isn't it?

Reply:
changed to make_plot